README.md

```diff
@@ -1,59 +1,40 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# My work portfolio
 
-> One paragraph statement about the project.
+> This project is a personal portfolio of my work. On Information Technology. It will be bould using HTML, css, and javascript. It also wil have a framework that is still to be defined. It could be bootstrap or something similar. I Will be creating all the code of this project on VScode and the repository will be on GitHub.
 
 ![screenshot](./app_screenshot.png)
 
 Additional description about the project and its features.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- Major languages: HTML, CSS, Javascript, GIT.
+- Frameworks: Bootstrap
+- Technologies used: mozilla, github, vscode, windows 10 and linux ubintu.
 
 ## Live Demo
 
-[Live Demo Link](https://livedemo.com)
+Will be delivered in the following steps of the project.
 
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
 To get a local copy up and running follow these simple example steps.
 
 ### Prerequisites
+A PC with a Windows 10 or Linux Ubuntu operating system. It has to have GIT installed.
 
 ### Setup
+Go to the following site and clone the repository (https://github.com/sergiogval/sergiogval.github.io)
+## Author
```

Comment:
- [ ] [OPTIONAL] Hey that would be better if you add your real name here to make your perfect readme BRILLILIANT 💎.